pkg/monitoring/rules/rules.go

```diff
@@ -0,0 +1,54 @@
+package rules
+
+import (
+	"github.com/machadovilaca/operator-observability/pkg/operatorrules"
+	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+
+	"kubevirt.io/containerized-data-importer/pkg/common"
+	"kubevirt.io/containerized-data-importer/pkg/monitoring/rules/alerts"
+	"kubevirt.io/containerized-data-importer/pkg/monitoring/rules/recordingrules"
+)
+
+const (
+	ruleName = "prometheus-cdi-rules"
+)
+
+// SetupRules initializes recording and alert rules in a namespace.
+func SetupRules(namespace string) error {
+	if err := recordingrules.Register(namespace); err != nil {
+		return err
+	}
+
+	if err := alerts.Register(namespace); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// BuildPrometheusRule creates a PrometheusRule in a namespace.
+func BuildPrometheusRule(namespace string) (*promv1.PrometheusRule, error) {
+	rules, err := operatorrules.BuildPrometheusRule(
+		ruleName,
+		namespace,
+		map[string]string{
+			common.CDIComponentLabel:  "",
```

Comment:
empty value?

Reply:
they did it like this before...

Reply:
see https://github.com/kubevirt/containerized-data-importer/blob/b33addca41bcc0eac76dc4cd37602f66b8cf0f71/pkg/operator/controller/prometheus.go#L285

Reply:
okay

Reply:
but I guess we can remove this maybe I'm not sure

Reply:
let's leave it as was